addons/mail/static/src/new/chat/chat_window.js

```diff
@@ -21,7 +20,7 @@ export class ChatWindow extends Component {
     }
 
     toggleFold() {
-        this.state.folded = !this.state.folded;
+        this.messaging.chatWindows.find(cw => cw.threadId === this.props.threadId).folded = !this.props.folded;
```

Comment:
Prettier is unhappy with this line of code
Make sure you have web/tooling properly configured.
This should not be possible without explicit `--no-verify` when making a commit.